packages/nextjs/components/ChallengeHeading.tsx

```diff
@@ -22,7 +30,7 @@ export function ChallengeHeading({ challengeId }: { challengeId: number }) {
           "text-primary": isCaptured,
         })}
       >
-        Challenge #{challengeId}
+        {SEASON_NAMES[season]} - Challenge #{challengeId}
```

Comment:
A couple of ideas to make it look a bit better (different symbols / sorting)

- `S1: Bangkok | Challenge 1`
- `Challenge #1 - Bangkok (S1)`

wdyt?

Reply:
Changed season name to "Bangkok (S1)" and using the second option as challenge title.